lib/basedriver/commands/timeout.js

```diff
@@ -11,10 +11,15 @@ let commands = {}, helpers = {}, extensions = {};
 const MIN_TIMEOUT = 0;
 
 commands.timeoutsW3C = async function (type, duration, scriptDuration, pageLoadDuration, implicitDuration) {
-  if (_.isNumber(implicitDuration)) {
-    return this.setImplicitWait(this.parseTimeoutArgument(implicitDuration));
-  } else if (_.isNumber(scriptDuration) || _.isNumber(pageLoadDuration)) {
-    throw new Error(`Not implemented yet for script and pageLoad.`);
+  if (_.isInteger(scriptDuration) || _.isInteger(pageLoadDuration) || _.isInteger(implicitDuration)) {
```

Comment:
I assume it could be rewritten as 

```
if (_.isInteger(scriptDuration) || _.isInteger(pageLoadDuration)) {
      throw new NotImplemntedError(`Not implemented yet for script and pageLoad.`);
}

if (_.isInteger(implicitDuration)) {
    return  this.setImplicitWait(this.parseTimeoutArgument(implicitDuration));
}

this.timeouts(type, duration);
```